apps/mobile/src/features/send-refactor/hooks/use-calculate-btc-max-spend.ts

```diff
@@ -0,0 +1,27 @@
+import { useCallback } from 'react';
+
+import { createCoinSelectionUtxos } from '@/features/send-refactor/utils';
+
+import { calculateMaxSpend } from '@leather.io/bitcoin';
+import { AverageBitcoinFeeRates } from '@leather.io/models';
+import type { Utxo } from '@leather.io/query';
+
+interface CalculateBtcMaxSpendParams {
+  recipient: string;
+  feeRate: number;
+}
+
+export type CalculateBtcMaxSpend = ReturnType<typeof useCalculateBtcMaxSpend>;
+
+export function useCalculateBtcMaxSpend(feeRates: AverageBitcoinFeeRates, utxos: Utxo[]) {
+  return useCallback(
```

Comment:
Does `useCallback` give a big benefit here? 

[Kent](https://kentcdodds.com/blog/usememo-and-usecallback) generally says to not bother

Reply:
The form is extremely dynamic with state updates propagating on each "key stroke" (whatever the mobile equivalent is). This limits the callback being recreated only when remote data changes. I'd say we probably need more `useCallback`s than fewer.